data_collection/web/guides.rst

```diff
@@ -40,8 +40,47 @@ Example::
 
 .. todo:: Explain how to change page URL and title for virtual page views or link to SPA instructions page.
 
+User ID
+-------
+
+User ID is an additional session dimension in yours analytics data. When set you will be able to aggregate data by this dimension. You can use Tag Manger, default Piwik Pro tag to setup `User ID` without editing any JS code. To set it manually follow this guide.
+
+To set up `User ID` dimension add a call to :ref:`setUserId<jtc-api-setUserId>` to your tracking code:
+
+.. code-block:: javascript
+
+    _paq.push(["setUserId", "user-name@example.com"]);
+
+.. note ::
+
+    #. `User ID` can't be longer than 1024 bytes, it will be 1024 characters unless you use only ASCII characters. When using Unicode check the length of user identifier, so you won't exceed 1024 bytes.
+    #. `User ID` should be a uniquie value for each user. Otherwise metrics for different users might be merged in the reports.
+    #. Usually a `User ID` value is an user email, because this is the identifier that users use to log in to a website and it fulfils above requirements.
+
+It is a good practice to remove value of `User ID`, when the user logs out. Otherwise `User ID` value might affect session of other users. To remove `User ID` value call :ref:`resetUserId<jtc-api-resetUserId>`.
```

Comment:
`value might affect session of other users.` + if they are sharing the device

Reply:
added